docs/getstarted/telemetry.md

```diff
@@ -25,22 +25,6 @@ If you use the JSON editor for your settings, add the following line:
     "telemetry.enableTelemetry": false
 ```
 
-## Disable crash reporting
```

Comment:
We should keep both sections, in FAQ and the telemetry topic. 
There may be external hard-coded links to code.visualstudio.com/docs/getstarted/telemetry#_disable-crash-reporting.
I don't think there is harm in having the same incantation in two places.

Reply:
updated, thanks!